config/spryk/spryks/Spryker/Glue/Dependency/Client/AddGlueDependencyClientDependencyProviderMethod.yml

```diff
@@ -0,0 +1,38 @@
+spryk: method
+description: "Adds a Client dependency method to the *module*DependencyProvider."
+mode: both
+
+arguments:
+  module:
+    inherit: true
+
+  dependentModule:
+    inherit: true
+
+  organization:
+    inherit: true
+    default: Spryker
+
+  providerMethod:
+    value: "add{{ dependentModule }}Client"
+
+  template:
+    value: Glue/Dependency/Client/GlueDependencyClientDependencyProviderMethod.php.twig
+
+  target:
+    value: "{{ organization }}\\Glue\\{{ module }}\\{{ module }}DependencyProvider"
+
+preSpryks:
+  - AddGlueDependencyProvider
+
+postSpryks:
```

Comment:
missing main provideDependencies function, missing using new function at the bottom of provideDependencies function